src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3128,6 +3095,39 @@ def update_load_balancer_profile(self, mc: ManagedCluster) -> ManagedCluster:
             )
         return mc
 
+    def update_outbound_type_in_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
If the implementation here is exactly the same as in azure-cli, may omit the func

Reply:
This function is not introduced in azure-cli 2.49